packages/trader/src/Modules/Trading/Containers/trade-params.jsx

```diff
@@ -11,27 +11,24 @@ import StopLoss from 'Modules/Trading/Components/Form/TradeParams/Multiplier/sto
 import TakeProfit from 'Modules/Trading/Components/Form/TradeParams/Multiplier/take-profit.jsx';
 import { connect } from 'Stores/connect';
 
-class TradeParams extends React.Component {
-    isVisible(component_key) {
-        return this.props.form_components.includes(component_key);
-    }
-
-    render() {
-        const { is_minimized } = this.props;
-        return (
-            <React.Fragment>
-                {this.isVisible('duration') && <Duration key={'duration'} is_minimized={is_minimized} />}
-                {this.isVisible('barrier') && <Barrier key={'barrier'} is_minimized={is_minimized} />}
-                {this.isVisible('last_digit') && <LastDigit key={'last_digit'} is_minimized={is_minimized} />}
-                {this.isVisible('amount') && <Amount key={'amount'} is_minimized={is_minimized} />}
-                {this.isVisible('multiplier') && <Multiplier key={'multiplier'} />}
-                {this.isVisible('take_profit') && <TakeProfit key={'take_profit'} />}
-                {this.isVisible('stop_loss') && <StopLoss key={'stop_loss'} />}
-                {this.isVisible('cancellation') && <CancelDeal key={'cancellation'} />}
-            </React.Fragment>
-        );
-    }
-}
+const TradeParams = props => {
```

Comment:
```suggestion
const TradeParams = ({is_minimized, form_components}) => {
```

Reply:
@reza-fs 
Thanks for comment 
code has been updated